docs/components/arm/xarmlite.md

```diff
@@ -4,11 +4,12 @@ linkTitle: "xArmLite"
 weight: 34
 type: "docs"
 description: "Configure an xArmLite arm."
+metaDescription: "Configure an xArmLite arm to integrate a UFACTORY Lite 6: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the xArmLite model. Click Create component. Edit and fill in the attributes as applicable."
```

Comment:
```suggestion
metaDescription: "Configure a UFACTORY Lite 6 arm: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the xArmLite model. Click Create component. Edit and fill in the attributes as applicable."
```

Reply:
I agree with this suggestion! Let's merge it in!